pkg/driver/utils.go

```diff
@@ -70,3 +72,9 @@ func addVolumeToTransitionList(volumeID string, status apis.CStorVolumeAttachmen
 	utils.TransitionVolList[volumeID] = status
 	return nil
 }
+
+// getCapacity converts capacity as string
+func getCapacity(cvc *apisv1.CStorVolumeConfig) string {
```

Comment:
n_p: This function is a good candidate of UT

Reply:
Needs to add some more tests, will add them in upcoming PRs